user-operator/src/main/java/io/strimzi/operator/user/model/KafkaUserModel.java

```diff
@@ -248,6 +251,18 @@ public void maybeGeneratePassword(PasswordGenerator generator, Secret userSecret
 
     }
 
+    /**
+     * Gerenate a SASL SSL Jaas Config String for SCRAM-SHA-512 secret
+     *
+     * @param username The KafkaPrincipal username
+     * @param password The SCRAM password for a given user.
+     * @return SASL jaas config string
+     */
+    public String saslJaasConfigBuilder(String username, String password) {
```

Comment:
I'd argue this isn't really a builder
a method name of `saslJaasConfig` is probably sufficient, you pass in user and pass and out comes a SASL jaas config string